''' 
FAULTLESS PYTHON PROGRAM 
Expects to pass Pylint and unittest run
'''


def addition_function_to_pass_unit_test(n1, n2):
    return (n1 + n2) 

if __name__ == "__main__":
    print(addition_function_to_pass_unit_test(5,5))
